GitCommands/DiffMergeTools/VsCode.cs

```diff
@@ -1,14 +1,18 @@
+using System;
 using System.Collections.Generic;
+using System.IO;
 
 namespace GitCommands.DiffMergeTools
 {
     internal class VsCode : DiffMergeTool
     {
+        private static readonly string[] Folders = GetFolders();
+
         /// <inheritdoc />
         public override string DiffCommand => "--wait --diff \"$LOCAL\" \"$REMOTE\"";
 
         /// <inheritdoc />
-        public override string ExeFileName => "code.exe";
+        public override string ExeFileName => "Code.exe";
```

Comment:
Why is this necessary?

Reply:
Not necessary, but this is what a code installs. I was tricked by this when searching for the executable. 

Note: I was not sure if @pmiossec wanted to change the expression or not. It seems to work now at least.